### PR TITLE
Sathursan vaccine add

### PR DESCRIPTION
**Pull request by SathursanS**
add and update vaccination screen

### CODE REVIEW

File: frontend/MainCode.js [hunk 1]
```diff
@@ -25,20 +27,29 @@ export default function MainCode() {
...
+    "Oxygen-Bold": require("./assets/fonts/Oxygen-Bold.ttf"),
+    "Inter-Light": require("./assets/fonts/Inter-Light.ttf"),
+    "Oxygen-Light": require("./assets/fonts/Oxygen-Light.ttf"),
   });
   const [popupAlert, setPopupAlert] = useState(false);
 
   return (
     loaded && (
       <>
-        <NavigationContainer>
```
Comment (RishanR) [inline frontend/MainCode.js:40]: Change this back to the NavigationContainer

File: frontend/screens/vaccinationScreens/addVaccinationRecord.js [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+import IconBadge from "../../components/iconBadge/custom-iconBadge";
+import Toast from "react-native-toast-message";
+
+const AddUpdateVaccinationRecords = ({ update, recordId }) => {
+  const token =
+    "eyJhbGciOiJSUzI1NiIsImtpZCI6IjM2NGU4NTQ1NzI5OWQ5NzIxYjczNDQyZGNiNTQ3Y2U2ZDk4NGRmNTkiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL3NlY3VyZXRva2VuLmdvb2dsZS5jb20vbXktaGVhbHRoLXBhbC1kZjQxYSIsImF1ZCI6Im15LWhlYWx0aC1wYWwtZGY0MWEiLCJhdXRoX3RpbWUiOjE2MjkwNzQ5NzEsInVzZXJfaWQiOiJ4UzZNTXV5WENDWTVlR05tS2RvZnRrY3B6MjMzIiwic3ViIjoieFM2TU11eVhDQ1k1ZUdObUtkb2Z0a2NwejIzMyIsImlhdCI6MTYyOTA3NDk3MSwiZXhwIjoxNjI5MDc4NTcxLCJlbWFpbCI6InJpc2hhbnJhdG5hQGhvdG1haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnsiZW1haWwiOlsicmlzaGFucmF0bmFAaG90bWFpbC5jb20iXX0sInNpZ25faW5fcHJvdmlkZXIiOiJwYXNzd29yZCJ9fQ.FSUq__5dhukERV3Zq_TdiKdqHRYNavsfibbOUL7RnL0x2jDhiPds8pARGQFaItwDpHR7Iet-UWztRoP0lZMSO_roVukYBWutC_lZdKM1a1mKzDs0Q3kV5bm0A5kZ7iT992AsMMM38EyPNoLom15wRa1oLH6MmAHd9LY6vpVSJj_FdxvWij0rlmYZB6_sF0DbMT2no6UgmYwwRkTzIg9A6YRFuH-z2MbAfIF8R11L_0zPTaB7Ubm9A3JW7R5_1GkQp8cCJu5HoHyXJAgRqMk-QqY12vF6e-YQvFoKRH41_MWyYhyenRhNXyrHe_HajvY3EC-PRcaynL-YfIkdmQKvVw";
```
Comment (RishanR) [inline frontend/screens/vaccinationScreens/addVaccinationRecord.js:19]: Remove the hard-coded token and perform a getToken() when sending the token to the api fetch to get the token from the SecureStore

**Review by RishanR — CHANGES_REQUESTED** [review]

File: frontend/MainCode.js (hunk 1 above)
Comment (RishanR) [reply frontend/MainCode.js:40]: woops i dont think i highlighted all the code properly here. Essentially under this line of code above you have some new code that you were using to test. Revert that back to the normal Navigation Code.

Comment (adshayanB) [reply frontend/MainCode.js:40]: +1

File: frontend/screens/vaccinationScreens/addVaccinationRecord.js (hunk 2 above)
Comment (adshayanB) [reply frontend/screens/vaccinationScreens/addVaccinationRecord.js:19]: Agreed, good catch!

File: frontend/MainCode.js (hunk 1 above)
Comment (adshayanB) [reply frontend/MainCode.js:40]: +1

File: frontend/screens/vaccinationScreens/addVaccinationRecord.js (hunk 2 above)
Comment (adshayanB) [reply frontend/screens/vaccinationScreens/addVaccinationRecord.js:19]: Agreed, good catch @RishanR

**Review by adshayanB — CHANGES_REQUESTED** [review]

**Review by SathursanS — COMMENTED** [review]
Will add the getToken when I make PR to hook up everything and fix DateTime component.

**Comment by adshayanB** [comment]
> Will add the getToken when I make PR to hook up everything and fix DateTime component.

I am not sure how effective that would be, if someone in our team decides to pull from master right?

File: frontend/MainCode.js [hunk 3]
```diff
@@ -39,11 +39,7 @@ export default function MainCode() {
   return (
     loaded && (
       <>
-        <AddUpdateVaccinationRecords
```
Comment (adshayanB) [inline frontend/MainCode.js]: LGTM!

**Review by adshayanB — APPROVED** [review]

**Pull request merged**